src/ircd/proto.rs

```diff
@@ -10,6 +10,8 @@ use tokio::net::TcpStream;
 use tokio::sync::mpsc;
 use tokio_util::codec::Framed;
 
+use crate::args::{args, AutoJoinOptions};
```

Comment:
AutoJoinOptions is unused since you've used the join_queries()/channels() methods